terracumber_config/tf_files/Uyuni-Master-NUE.tf

```diff
@@ -151,7 +151,7 @@ module "cucumber_testsuite" {
       install_salt_bundle = true
     }
     suse-minion = {
-      image = "sles15sp2o"
+      image = "sles15sp3o"
```

Comment:
Let's do something, because this is confusing.
Let's add a comment and explain that we use a previous SP, so we upgrade it inside the test suite. Make sense?

Reply:
Good idea.

Reply:
Added.